### ISSUE TITLE
interested in a PR?

### ISSUE THREAD

**Issue by bbolker** [bcjaeger/r2glmm#15]
I made a bunch of [small changes](https://github.com/bbolker/r2glmm/commits/master); it's a little bit messy and you might not want all of it.  The most important change is that I added some support for `gamm` objects from the `gamm4` package ...  I could probably create a new branch and cherry-pick the useful stuff if you preferred ...

**Comment by bcjaeger** [comment]
Absolutely! The edits look very helpful. It would be great to use a new branch containing the changes you feel are most important. I should be able to review that much faster. Thank you so much for your interest in improving r2glmm.
